src/testUtil/TestCase.ts

```diff
@@ -139,8 +140,20 @@ export class TestCase {
   }
 
   private getExcludedFields(context?: { initialSnapshot?: boolean }) {
+    const clipboardActions: ActionType[] = context?.initialSnapshot
+      ? ["pasteFromClipboard"]
+      : ["copyToClipboard", "cutToClipboard"];
+
+    const visibleRangeActions: ActionType[] = [
+      "foldRegion",
+      "unfoldRegion",
+      "scrollToBottom",
+      "scrollToCenter",
+      "scrollToTop",
+    ];
```

Comment:
These had rotted when we upgraded action names.  I added explicit types to prevent that happening in the future